hugo.yaml

```diff
@@ -89,25 +90,25 @@ params:
         improve</a>.
   links: # CUSTOMIZE
     user:
-      - name: GitHub Discussions
-        url: https://github.com/google/docsy/discussions
+      - name: GitHub community
+        url: https://github.com/theupdateframework/community
         icon: fa-brands fa-github
         desc: Discussion and help from your fellow users
       - name: User mailing list
-        url: https://groups.google.com/forum/#!forum/docsy-users
+        url: https://groups.google.com/forum/?fromgroups#!forum/theupdateframework
         icon: fa-solid fa-envelope
-        desc: Sign up for Docsy announcements
-      - name: Twitter
-        url: https://twitter.com/docsydocs
-        icon: fa-brands fa-x-twitter
-        desc: Follow us on Twitter to get the latest news!
-      - name: Stack Overflow
-        url: https://stackoverflow.com/questions/tagged/docsy
-        icon: fa-brands fa-stack-overflow
-        desc: Practical questions and curated answers
+        desc: Sign up for TUF announcements
+      - name: Slack channel
+        url: https://communityinviter.com/apps/cloud-native/cncf
+        icon: fa-brands fa-slack
+        desc: Connect with us on CNCF slack channel tuf!
+      - name: Community meetings
+        url: https://www.cncf.io/calendar
+        icon: fa-regular fa-calendar-days
+        desc: Join our community meetings
```

Comment:
I'm not seeing any meetings for TUF in that calendar page. Am I missing something?

Reply:
I'm also not sure. I found it listed on TUF community page as their calendar https://github.com/theupdateframework/community?tab=readme-ov-file.

 Maybe @lukpueh can elaborate

Reply:
Oh, for now, just link to that page. (We can change it later if/as needed.)